aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -0,0 +1,179 @@
+# Controller Service
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ include "aws-ebs-csi-driver.fullname" . }}-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "aws-ebs-csi-driver.controller.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.controller.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.controller.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "aws-ebs-csi-driver.controller.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "aws-ebs-csi-driver.controller.selectorLabels" . | nindent 8 }}
+      annotations:
+        {{- toYaml .Values.controller.podAnnotations | nindent 8 }}
+    spec:
+      nodeSelector:
+        {{- toYaml .Values.controller.nodeSelector | nindent 8 }}
+      serviceAccountName: {{ include "aws-ebs-csi-driver.controller.serviceAccountName" . }}
+      priorityClassName: {{ .Values.controller.priorityClassName }}
+      affinity:
+        {{- toYaml .Values.controller.affinity | nindent 8 }}
+      tolerations:
+        {{- toYaml .Values.controller.tolerations | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-controller
+          image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - controller
+            - --endpoint=$(CSI_ENDPOINT)
+            - --logtostderr
+            - --v={{ .Values.controller.logLevel }}
+            {{- with .Values.controller.extraArgs }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+            {{- if .Values.extraVolumeTags }}
+              {{- include "aws-ebs-csi-driver.extra-volume-tags" . | nindent 12 }}
+            {{- end }}
+          env:
+            - name: CSI_ENDPOINT
+              value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: aws-secret
+                  key: key_id
+                  optional: true
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: aws-secret
+                  key: access_key
+                  optional: true
```

Comment:
@cw-sakamoto 
Isn't `Secret` generation a part of this `Chart`?

Reply:
@cw-ozaki 
I fixed in https://github.com/chatwork/charts/pull/196/commits/1ce2d8e42ca77345aedfbd55520ab0bbe1c398a3.
When the existing one is specified in values, it is not created, but when `keyId` and `accessKey` are written in values, it is created.

Reply:
@cw-sakamoto 
I've checked the fix! Thank you!
It is better to set the hash of `Secret` in the `Pod` label.